internals/secrethub/env_ls.go

```diff
@@ -0,0 +1,50 @@
+package secrethub
+
+import (
+	"fmt"
+
+	"github.com/secrethub/secrethub-cli/internals/cli/ui"
+	"github.com/secrethub/secrethub-cli/internals/secrethub/command"
+)
+
+// EnvListCommand is a command to list all environment variable keys set in the process of `secrethub run`.
+type EnvListCommand struct {
+	io          ui.IO
+	environment *environment
+}
+
+// NewEnvListCommand creates a new EnvListCommand.
+func NewEnvListCommand(io ui.IO) *EnvListCommand {
+	return &EnvListCommand{
+		io:          io,
+		environment: newEnvironment(io),
+	}
+}
+
+// Register adds a CommandClause and it's args and flags to a Registerer.
+func (cmd *EnvListCommand) Register(r command.Registerer) {
+	clause := r.Command("ls", "Read the value of a single environment variable.")
+	clause.Alias("list")
+
+	cmd.environment.register(clause)
+
+	command.BindAction(clause, cmd.Run)
+}
+
+// Run handles the command with the options as specified in the command.
```

Comment:
Super tiny thing and not required to fix (now), but saw your PR and was curious and scrolled past this piece of documentation. Consider putting something a bit simpler here as this sentence contains a few abstract concepts that can add more noise than they explain.

What is the key thing you need to understand about this function? Consider:

```go
// Run executes the command. This is where the actual logic of the command lives.
```
Or even simpler:
```go
// Run executes the command. 
```